### ISSUE TITLE
My website is banned...

### ISSUE THREAD

**Issue by DevScyu** [surgeplay/Visage#4]
So my website is banned from using Visage... There were no warnings or anything. 
(this is the only way i know how to contact you sorry if its the wrong place to put it)

**Comment by exaskye** [comment]
You _were_ warned. I told you a month ago that you needed to fix your request volume, and that searching for "thun" returns ~1000 responses that then result in ~1000 simultaneous Visage requests. I linked you a lazy loading image library, and you said you would look into implementing it "tomorrow" and I never heard another word from you.

**Comment by DevScyu** [comment]
I implimented a UUID fetcher into the website so all the requests should be fine right? You also did not warn me of my website getting banned from your services.

Also lazy load is in effect ;-;

**Comment by exaskye** [comment]
It looks like you included the lazyload library but don't use it. I just confirmed again that searching for "thun" results in 1754 simultaneous requests to Visage.

UUIDs are better than usernames, but requesting ~2k simultaneous renders is still no good.

EDIT: Actually, it looks like you only execute it on page load, so it doesn't catch the new img tags you add after doing a search. If you call the lazyloader again after searching it may fix it.

I've also switched the ban to a ratelimit instead.
